test/pollfd.jl

```diff
@@ -1,52 +1,90 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-@unix_only begin
```

Comment:
why did you remove this? msvcrt has `_pipe` but it has extra arguments: https://msdn.microsoft.com/en-us/library/aa298531(v=vs.60).aspx